README.md

```diff
@@ -156,11 +156,18 @@ GPU and nightly builds are available in our public GCS bucket.
 | nightly (Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly-cp310-cp310-linux_x86_64.whl` |
 | nightly (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-nightly-cp38-cp38-linux_x86_64.whl` |
 
-You can also add `+yyyymmdd` after `torch_xla-nightly` to get the nightly wheel of a specified date. Here is an example:
+To install the release candidate, you can replace `torch_xla-nightly` with `torch_xla-2.4.0rc1`. Similarly, to install the corresponding Torch version, simply replace `torch_xla` with `torch`.
+
+For installing a nightly wheel of a specified date, you can append `+yyyymmdd` after `torch_xla-nightly` or `torch-nightly`. To obtain the corresponding `libtpu` version, you can use the following link to install the desired wheels:
+
+`https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/wheels/libtpu-nightly/libtpu_nightly-0.1.devyyyymmdd-py3-none-any.whl`.
+
+Here is an example:
 
 ```
 pip3 install torch==2.5.0.dev20240613+cpu --index-url https://download.pytorch.org/whl/nightly/cpu
 pip3 install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly%2B20240613-cp310-cp310-linux_x86_64.whl
+pip3 install https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/wheels/libtpu-nightly/libtpu_nightly-0.1.dev20240613-py3-none-any.whl
```

Comment:
you only need to run `pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html` which will install the correct version of the libtpu based on the pytorch/xla installed. We also mentioned that in the begining of this part but maybe that's not clear enough.

Reply:
@JackCaoG Could you please confirm if I am correct with the following statement? 

To install the stable version of `torch_xla`, we use the command:
```bash
pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
```

However, I noticed that there are no instructions in the documentation for installing `libtpu` when installing the nightly version of `torch_xla`. 

Is it possible to use `torch_xla[tpu]` in a similar manner for the nightly version as well?

Reply:
`pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html` will always install the libtpu version ping by the current torch_xla that's installed in the system. 

Each torch_xla pin the libtpu version in https://github.com/pytorch/xla/blob/master/setup.py#L68